ci_framework/roles/edpm_prepare/tasks/main.yml

```diff
@@ -101,6 +101,11 @@
           --namespace={{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }}
           --for=jsonpath='{.status.phase}'=Complete --timeout=20m
 
+    - name: Wait for openstack operator to be deployed
```

Comment:
Can I ask why we do this if already wait on line 94?

Reply:
I check with Fabricio, `make wait` is added for operator specific installation. But these waits`make_openstack_wai` are added for deployment.

Reply:
I check with Fabricio, `make wait` is added for operator specific installation. But these waits`make_openstack_wai` are added for deployment.

Reply:
@pablintino Thank you for reviewing it, the goal is of this utility is to replace this https://github.com/openstack-k8s-operators/ci-framework/pull/178 pr with above new target.

It will help to find out the issues after install plan and to know whether operator installation is successful.